test/integration/main_test.go

```diff
@@ -110,13 +110,13 @@ func TestMain(m *testing.M) {
 
 // TestMappingEventHandling tests the entire flow from uploading a mapping data
 // to GCS, triggering pmap event handler, to writing it to a BigQuery table.
-// TODO(#56): add validation logic for github source attributes as object metadata.
```

Comment:
Add a comment here 
// TODO(#185): Refactor so we don't have to enforce subscope in every test case.

Reply:
Added